sdk/textanalytics/azure-ai-textanalytics-perf/src/main/java/com/azure/ai/textanalytics/perf/DetectLanguageTest.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.perf;
+
+import com.azure.ai.textanalytics.models.DetectedLanguage;
+import com.azure.ai.textanalytics.perf.core.ServiceTest;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Mono;
+
+/**
+ * Performs custom model recognition operations.
+ */
+public class DetectLanguageTest extends ServiceTest<PerfStressOptions> {
+    /**
+     * The CustomModelRecognitionTest class.
+     *
+     * @param options the configurable options for perf testing this class
+     */
+    public DetectLanguageTest(PerfStressOptions options) {
+        super(options);
+    }
+
+    @Override
+    public void run() {
+        final DetectedLanguage detectedLanguage = textAnalyticsClient.detectLanguage(
+            "Detta är ett dokument skrivet på engelska.");
+        assert "swedish".equals(detectedLanguage.getName());
+    }
+
+    @Override
+    public Mono<Void> runAsync() {
+        return textAnalyticsAsyncClient.detectLanguage("Detta är ett dokument skrivet på engelska.")
```

Comment:
You need to send a bunch of documents and make the default to be 1k which is the max the service accepts for this API.